member.py

```diff
@@ -1045,10 +1021,11 @@ def get(self):
             return # todo figure out what to do
 
         the_member_key=ndb.Key(urlsafe=the_member_keyurl)
-        
-        the_user = self.user # todo - make sure the user is a superuser
-        
-        if (the_user != the_member_key):
+
+        if not member_is_superuser(self.user):
+            self.abort_non_admin()
+
+        if (self.user != the_member_key):
             forget_member_from_key(the_member_key)
         else:
             print 'cannot delete yourself, people'
```

Comment:
Should probably throw an exception. The only way to get here is to manually paste your key into the url - someone doing that is a troublemaker.